pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,8 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
+	// Since the APIGroup may not contain a '/', you can get the APIGroup by parsing from the last '/'
```

Comment:
that assumes APIPrefix doesn't contain a slash either... is that a valid assumption?